eta/modules/sample_videos.py

```diff
@@ -37,6 +37,7 @@
 import eta.core.image as etai
 import eta.core.module as etam
 import eta.core.video as etav
+import eta.core.utils as etau
```

Comment:
nit: style says alphabetize imports. 

ETA has a style guide and we follow it https://github.com/voxel51/eta/blob/develop/docs/python_style_guide.md.